### DISCUSSION TITLE
Linked record deletion [Jan 20]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4953] [Discord Threads]
> 🙋‍♂️ **Jakob from bunch**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Jakob from bunch**  
#### <@857998597474484295> maybe?  
#  
> 🙋‍♂️ **Dstala | NocoDB Committer**  
#### <@1006611047198761032> currently, links have to be removed first before deleting the record.    
#### let me check with the team if there is an easier alternative  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1054688140289376266)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
